analyzer/metadata-plugin/src/main/java/eu/fasten/analyzer/metadataplugin/MetadataDatabasePlugin.java

```diff
@@ -118,6 +120,9 @@ public void consume(String topic, ConsumerRecord<String, String> record) {
                         }
                     });
                 } catch (Exception expected) {
+                    logger.error("An exception occurred {} ", expected.getStackTrace());
```

Comment:
There is no need for these 3 lines. An exception is already logged earlier in the transaction. It is just re-thrown to rollback the transaction, so we need an empty catch block to catch it. I will remove there 3 lines (121-123). Don't worry, this transactional error handling is quite confusing :)

Reply:
Okay, I will remove it.